### PR TITLE
dbus: add ConnectTo method

### PR DESCRIPTION
**Pull request by metafex**
This adds a ConnectTo method to the dbus part so one can connect to different dbus deamons. My usecase for this is to connect to remote servers via socat and ssh to control systemd.

### CODE REVIEW

File: dbus/dbus.go [hunk 1]
```diff
@@ -92,13 +98,27 @@ func New() (*Conn, error) {
...
 	if err != nil {
-		return err
+		return nil, err
+	}
+
+	if err = c.initConnection(); err != nil {
+		return nil, err
 	}
 
+	c.initJobs()
```
Comment (bcwaldon) [inline dbus/dbus.go]: Why is `initjobs` getting called in `ConnectTo`, but not `New` ?

**Comment by bcwaldon** [comment]
@metafex sorry for letting this sit so long. Could you rebase this on top of current master?

**Comment by philips** [comment]
ping @metafex

**Comment by philips** [comment]
Closing until @metafex can rebase this on top of master.

**Pull request closed without merging**